pkg/handler/v2/cluster/cluster_test.go

```diff
@@ -1144,6 +1146,193 @@ func TestGetClusterHealth(t *testing.T) {
 	}
 }
 
+func TestGetClusterMetrics(t *testing.T) {
+	t.Parallel()
+	cpuQuantity, err := resource.ParseQuantity("290")
+	if err != nil {
+		t.Fatal(err)
+	}
+	memoryQuantity, err := resource.ParseQuantity("687202304")
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	testcases := []struct {
+		Name                   string
+		Body                   string
```

Comment:
NIT: Body is not needed, you can just use `nil` for the reader or hardcode an empty string here:
```
req := httptest.NewRequest("GET", fmt.Sprintf("/api/v2/projects/%s/clusters/%s/metrics", test.ProjectName, tc.ClusterToGet), strings.NewReader(tc.Body))

```